system/jhtree/jhtree.cpp

```diff
@@ -673,6 +670,27 @@ class CNodeMRUCache final : public CMRUCacheOf<CKeyIdAndPos, CNodeCacheEntry, CN
     std::atomic<size32_t> sizeInMem{0};
     size32_t memLimit = 0;
 public:
+    std::shared_ptr<hpccMetrics::CounterMetric> numHits = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numAdds = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numDups = nullptr;
+    std::shared_ptr<hpccMetrics::CounterMetric> numEvicts = nullptr;
```

Comment:
would a CRuntimeStatisticCollection with a mapping based on StatisticKind's wouldn't be more consistent with other code? (and allow for copying/deltas etc.)

Reply:
Not sure what you mean. The purpose of this PR is to expose these values via metrics interface.